### PR TITLE
GPC-NONE: Add in trivy

### PR DESCRIPTION
**Pull request by oskarwilliams378**

### CODE REVIEW

File: .trivyignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## DOCKER
+
+# ADD is used for getting file from the internet
+AVD-DS-0005
+# Healthcheck already exists
+AVD-DS-0026
```
Comment (olevett) [inline .trivyignore:6]: I don't think is is true - we have a healthcheck in the docker-compose and in ECS config, but we don't have one in our dockerfile itself

**Comment by olevett** [comment]
What gap is this filling out of interest, is it earlier/different coverage to e.g. ECS scanning?

File: pyproject.toml [hunk 2]
```diff
@@ -1,17 +0,0 @@
-[tool.poetry]
```
Comment (olevett) [inline pyproject.toml:1]: Is this not used by the lambdas?

File: .trivyignore (hunk 1 above)
Comment (oskarwilliams378) [reply .trivyignore:6]: Will update

**Comment by oskarwilliams378** [comment]
This will fill the gap of ECR image scanning, I'll add a daily job for this afterwards. It also allows us to remove snyk. Basically it gets us down to one system in one place which is quite nice!

**Review by olevett — APPROVED** [review]

**Pull request merged**